src/app/features/test-records/components/test-amendment-history/test-amendment-history.component.scss

```diff
@@ -0,0 +1,5 @@
+@media (min-width: 40.0625em) {
+  h2 {
+    margin-top: 30px;
```

Comment:
Take this with a pinch of salt, but is there maybe a neater way of doing this? Wrap the table above inside a container/section or something? assume that is what it is attempting to do is create margin above the title